LabZoo/Animal.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace LabZoo
+{
+    public class Animal
```

Comment:
Not sure how `Program` ended up renamed as `Animal`, but I'd rename and move this `class` back to `Program.cs`.